services/src/main/java/org/exoplatform/webconferencing/rest/RESTWebConferencingService.java

```diff
@@ -639,21 +639,30 @@ public Response getRoomInfo(@Context UriInfo uriInfo,
 
   @GET
   @RolesAllowed("users")
-  @Path("{spaceId}/providers")
+  @Path("{identityId}/providers")
   @Operation(summary = "Retrieves the list of active providers for space", method = "GET", description = "Retrieves the list of active providers for space")
   @ApiResponses(value = { @ApiResponse(responseCode = "200", description = "Request fulfilled"),
       @ApiResponse(responseCode = "401", description = "Unauthorized"),
       @ApiResponse(responseCode = "500", description = "Internal server error"), })
-  public Response getActiveProvidersForSpace(@Parameter(description = "Space Id", required = true)
-  @PathParam("spaceId")
-  String spaceId) {
+  public Response getActiveProviders(@Parameter(description = "isSpace", required = true)
+  @QueryParam("isSpace")
+  boolean isSpace,
```

Comment:
we should continue to use identityId as a real identityId and not a spacePrettyName
With this option, there is no need to use parameter isSpace and isUser